socorro/external/es/create_recent_indices_app.py

```diff
@@ -41,7 +47,10 @@ def main(self):
         today = date.today()
         current_monday = today - timedelta(days=today.weekday())
 
-        for week_diff in range(-2, self.config.elasticsearch_weeks_to_create):
+        past = self.config.weeks_to_create_past * -1
+        future = self.config.weeks_to_create_future
+
+        for week_diff in range(past, future):
```

Comment:
Previously, you'd specify "create only 1 week", but it'd create 3 which makes no sense. I'm pretty sure that's my fault. This splits it into two arguments and lets you adjust accordingly.